dev

```diff
@@ -88,4 +92,9 @@ clean() {
     rm $MARKER
 }
 
+# view logs
+logs() {
+    docker compose logs $1 wordpress
+}
```

Comment:
What is this doing...., oh I get it now.
The `$1` is the options param like `-f` or `--tail`
And then the service is wordpress, naturally.

https://docs.docker.com/engine/reference/commandline/compose_logs/